TACC/Core-Portal
# task/FP-591 - Fix Jobs timezone display of job creation times

## Overview: ##

- Fix job created time

## PR Status: ##


## Related Jira tickets: ##

* [FP-591](https://jira.tacc.utexas.edu/browse/FP-591)

## Summary of Changes: ##

- Jobs will use formatters in utils/timeFormat
- utils/timeFormat manually applies the timezone offset to the time, since toLocaleTimeString functions do not seem to honor the timezone offset returned by Jobs

## Testing Steps: ##
1. Submit a job
2. Verify that the job creation time matches your local time

## Notes ##

Data files seem to have the correct timestamps, so it will not use the utils/timeFormat functions.
